synergy-design-system/synergy-design-system
# feat: ✨ syn-spinner

# Pull Request

## 📖 Description

This PR adds the new `<syn-spinner />` element.

### 🎫 Issues

Closes #379

## 👩‍💻 Reviewer Notes

Just have a look at the provided Storybook documentation.

## 📑 Test Plan

- Check out this branch and build world.
- Spin up storybook
- Compare the original Components -> Button -> Loading Story to make sure the changes do not break existing features
- Have a look at the new Components -> Spinner stories

## ✅ DoD
